### PR TITLE
docs(legacy-migrations): updated legacy migration guides

### PR DESCRIPTION
**Pull request by daragh-king-genesys**
updated legacy migration guides

COMUI-2497

### CODE REVIEW

File: docs/migration-guides/v4/readme.md [hunk 1]
```diff
@@ -142,20 +142,20 @@ If possible, avoid the usage of legacy components that have a migration path and
...
-| `gux-side-panel-legacy`      | N/A                                         | N/A                                                    |
-| `gux-tabs-legacy`            | `gux-tabs`, gux-tabs-advanced`              | [migration details](../v3/gux-tabs-legacy.md)          |
-| `gux-text-label-legacy`      | `gux-form-field-{type}`                     | [migration details](../v3/gux-form-field-legacy.md)    |
+| V3 tag name                  | V4 stable equivalent (requires API changes) | Migration Guide                                    |
+| ---------------------------- | ------------------------------------------- | -------------------------------------------------- |
+| `gux-accordion-legacy`       | `gux-accordion`                             | [migration details](./gux-accordion-legacy.md)     |
+| `gux-action-button-legacy`   | `gux-action-button`                         | [migration details](./gux-action-button-legacy.md) |
+| `gux-action-list-legacy`     | `gux-list`                                  | [migration details](./gux-action-button-legacy.md) |
```
Comment (thomas-c-dillon) [inline docs/migration-guides/v4/readme.md:149]: The link to the migration guide is going to the wrong component. Copy/paste error?

**Review by thomas-c-dillon — CHANGES_REQUESTED** [review]
I didn't test the migration steps for each components but it is looking pretty good to me. The issues for the action button and button multi have been included!

**Review by jason-evans-genesys — APPROVED** [review]
Other than what @thomas-c-dillon mentioned looks good to me!

File: docs/migration-guides/v4/readme.md (hunk 1 above)
Comment (daragh-king-genesys) [reply docs/migration-guides/v4/readme.md:149]: No the `gux-action-list-legacy` is a sub component of `gux-action-button-legacy`.

**Pull request merged**